cli/internal/runcache/runcache.go

```diff
@@ -170,9 +174,7 @@ func (tc TaskCache) RestoreOutputs(terminal *cli.PrefixedUi, logger hclog.Logger
 	case util.FullTaskOutput:
 		logger.Debug("log file", "path", tc.LogFileName)
 		if tc.LogFileName.FileExists() {
-			// The task label is baked into the log file, so we need to grab the underlying Ui
-			// instance in order to not duplicate it
-			tc.rc.logReplayer(logger, terminal.Ui, tc.LogFileName)
+			tc.rc.logReplayer(logger, terminal, tc.LogFileName)
```

Comment:
I may have missed something, but it looks like the comment that was deleted here still applies? `terminal` passed in above is from `NewTerminal`, which is prefixed.

Reply:
I think this is still the case. The log files currently contain a prefix. So depending on the value of the new flag, you are going to end up duplicating the prefix.